api/estados/santacatarina.js

```diff
@@ -1,300 +1,302 @@
-  "São José do Cerrito",
-  "São Lourenço do Oeste",
-  "São Ludgero",
-  "São Martinho",
-  "São Miguel da Boa Vista",
-  "São Miguel do Oeste",
-  "São Pedro de Alcântara",
-  "Saudades",
-  "Schroeder",
-  "Seara",
-  "Serra Alta",
-  "Siderópolis",
-  "Sombrio",
-  "Sul Brasil",
-  "Taió",
-  "Tangará",
-  "Tigrinhos",
-  "Tijucas",
-  "Timbé do Sul",
-  "Timbó",
-  "Timbó Grande",
-  "Três Barras",
-  "Treviso",
-  "Treze de Maio",
-  "Treze Tílias",
-  "Trombudo Central",
-  "Tubarão",
-  "Tunápolis",
-  "Turvo",
-  "União do Oeste",
-  "Urubici",
-  "Urupema",
-  "Urussanga",
-  "constgeão",
-  "constgem",
-  "constgem Bonita",
-  "Vidal Ramos",
-  "Videira",
-  "Vitor Meireles",
-  "Witmarsum",
-  "Xanxerê",
-  "Xavantina",
-  "Xaxim",
-  "Zortéa"
-  ]
+const sc = {
+  state: 'Santa Catarina',
+  abbreviation: 'sc',
+  cities: [
+    'Abdon Batista',
+    'Abelardo Luz',
+    'Agrolândia',
+    'Agronômica',
+    'Água Doce',
+    'Águas de Chapecó',
+    'Águas Frias',
+    'Águas Mornas',
+    'Alfredo Wagner',
+    'Alto Bela Vista',
+    'Anchieta',
+    'Angelina',
+    'Anita Garibaldi',
+    'Anitápolis',
+    'Antônio Carlos',
+    'Apiúna',
+    'Arabutã',
+    'Araquari',
+    'Araranguá',
+    'Armazém',
+    'Arroio Trinta',
+    'Arvoredo',
+    'Ascurra',
+    'Atalanta',
+    'Aurora',
+    'Balneário Arroio do Silva',
+    'Balneário Barra do Sul',
+    'Balneário Camboriú',
+    'Balneário Gaivota',
+    'Bandeirante',
+    'Barra Bonita',
+    'Barra Velha',
+    'Bela Vista do Toldo',
+    'Belmonte',
+    'Benedito Novo',
+    'Biguaçu',
+    'Blumenau',
+    'Bocaina do Sul',
+    'Bom Jardim da Serra',
+    'Bom Jesus',
+    'Bom Jesus do Oeste',
+    'Bom Retiro',
+    'Bombinhas',
+    'Botuverá',
+    'Braço do Norte',
+    'Braço do Trombudo',
+    'Brunópolis',
+    'Brusque',
+    'Caçador',
+    'Caibi',
+    'Calmon',
+    'Camboriú',
+    'Campo Alegre',
+    'Campo Belo do Sul',
+    'Campo Erê',
+    'Campos Novos',
+    'Canelinha',
+    'Canoinhas',
+    'Capão Alto',
+    'Capinzal',
+    'Capiconsti de Baixo',
+    'Catanduvas',
+    'Caxambu do Sul',
+    'Celso Ramos',
+    'Cerro Negro',
+    'Chapadão do Lageado',
+    'Chapecó',
+    'Cocal do Sul',
+    'Concórdia',
+    'Cordilheira Alta',
+    'Coronel Freitas',
+    'Coronel Martins',
+    'Correia Pinto',
+    'Corupá',
+    'Criciúma',
+    'Cunha Porã',
+    'Cunhataí',
+    'Curitibanos',
+    'Descanso',
+    'Dionísio Cerqueira',
+    'Dona Emma',
+    'Doutor Pedrinho',
+    'Entre Rios',
+    'Ermo',
+    'Erval Velho',
+    'Faxinal dos Guedes',
+    'Flor do Sertão',
+    'Florianópolis',
+    'Formosa do Sul',
+    'Forquilhinha',
+    'Fraiburgo',
+    'Frei Rogério',
+    'Galvão',
+    'Garopaba',
+    'Garuva',
+    'Gaspar',
+    'Governador Celso Ramos',
+    'Grão Pará',
+    'Gravatal',
+    'Guabiruba',
+    'Guaraciaba',
+    'Guaramirim',
+    'Guarujá do Sul',
+    'Guatambú',
+    "Herval d'Oeste",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)